addons/airway/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -65,6 +65,17 @@ class ACE_Medical_Treatment_Actions {
         callbackProgress = "";
         sounds[] = {{QPATHTO_R(sounds\accuvac.wav),6,1,15}};
     };
+    class Suction: Larynxtubus {
```

Comment:
i would inherit from accuvac class so you dont have to repeat those "methods" below that are same as accuvac

Reply:
Ok...
Now inheriting.
and I changed the order of class to reduce the number of lines of code as much as possible.

Reply:
it's solved